packages/pyconfigure.rb

```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Pyconfigure < Package
+  description 'GNU pyconfigure provides developers with file templates for implementing standard configure scripts and Makefile recipes for their Python packages.'
+  homepage 'https://www.gnu.org/software/pyconfigure/'
+  version '3.1.1'
+  compatibility 'all'
+  source_url 'https://ftpmirror.gnu.org/pyconfigure/pyconfigure-0.2.3.tar.gz'
+  source_sha256 '37438ee1ae0aad7692188c7d01218cb407a1c89f3efdb5d360311b7ade4ab14f'
+  
+  depends_on 'python2_7'
```

Comment:
`depends_on 'python27'`?

Reply:
Actually, you can just exclude since it's a core package.

Reply:
done